telegram/files/animation.py

```diff
@@ -26,6 +26,9 @@ class Animation(TelegramObject):
 
     Attributes:
         file_id (:obj:`str`): Unique file identifier.
```

Comment:
From https://core.telegram.org/bots/api#animation
```file_id - Identifier for this file, which can be used to download or reuse the file```
I think we should use that here too. (also for all the others)